src/calibrationlib/operator_feature_exporter.hpp

```diff
@@ -0,0 +1,115 @@
+#pragma once
+
+#include <string>
+
+#include <operators/abstract_operator.hpp>
+#include "csv_writer.hpp"
+
+namespace opossum {
+
+class OperatorFeatureExporter {
+ public:
+  explicit OperatorFeatureExporter(const std::string& path_to_dir);
+
+  void export_to_csv(std::shared_ptr<const AbstractOperator> op) const;
```

Comment:
const

Reply:
Added const to all std::shared_ptr's for the class OperatorFeatureExporter